tests/e2e/configs/run-watsonx.yaml

```
@@ -142,6 +145,11 @@ registered_resources:
   - shield_id: llama-guard
     provider_id: llama-guard
     provider_shield_id: openai/gpt-4o-mini
+  vector_stores: 
+  - embedding_dimension: 768
+    embedding_model: sentence-transformers/sentence-transformers/all-mpnet-base-v2
+    provider_id: faiss
+    vector_store_id: ${env.FAISS_VECTOR_STORE_ID}
   vector_stores: []
```

YAMLlint (1.38.0)

[error] duplication of key "vector_stores" in mapping

(key-duplicates)